devicemodel/hw/pci/virtio/virtio_block.c

```diff
@@ -506,7 +506,7 @@ virtio_blk_init(struct vmctx *ctx, struct pci_vdev *dev, char *opts)
 		if (strcmp("iothread", opt) == 0) {
 			use_iothread = true;
 		} else {
-			opts_tmp = opts_start;
+			opts_tmp = opts;
```

Comment:
Can some comments be added?
It seems that the opts_start is assigned by using strdup(opts). 
This seems quite strange. It will be better to add some comments about the buffer pointed by opts_start is already changed.